src/Pages/ResourcesPage.jsx

```diff
@@ -67,7 +67,7 @@ const housingFacts = [
         src: dream,
         title: "Private Housing",
         textList: [
-            "Landlords have full disccretion, and often deny people with criminal records",
+            "Landlords have full discretion, and often deny people with criminal records",
```

Comment:
Looks like there's also a typo in the word "applications" on line 43, and an extra space on line 88